wcs2:post-xml-req5 fails with InvocationTargetException (post/xml)

## Setup
SUT: TODO
Selected CCs: all, except _WCS WPS 2.0 (Processing)_
Tested locally with docker environment 
  * ETS WCS 2. 0 1.13-SNAPSHOT 
 * TEAM Engine 5.2-SNAPSHOT

## Testfailure
"Error in call to extension function {public org.w3c.dom.NodeList com.occamlab.te.TECore.request(org.w3c.dom.Document,java.lang.String) throws java.lang.Throwable}: Exception in extension function java.lang.reflect.InvocationTargetException"

* Service returns a multipart response. ETS fails with the following failure (error_log.txt):
```
Caused by: java.lang.RuntimeException: Failed to parse resource from https://ogc.webgistesting.net/server/services/Mynetcdf/ImageServer/WCSServer 
 Failed to parse input: sun.net.www.protocol.http.HttpURLConnection$HttpInputStream
        at com.occamlab.te.parsers.XMLValidatingParser.parse(XMLValidatingParser.java:222)
        ... 83 more
```

## Proposal
 * Further investigations required

--- a/src/main/scripts/ctl/ext_post-xml.xml
+++ b/src/main/scripts/ctl/ext_post-xml.xml
@@ -273,25 +273,20 @@
 									xsi:schemaLocation="http://www.opengis.net/wcs/2.0 ../../wcsAll.xsd"
 									service="WCS" version="2.0.1">
 									<wcs:CoverageId><xsl:value-of select="$Capabilities//*[local-name()='CoverageSummary'][1]/*[local-name()='CoverageId']"/></wcs:CoverageId></wcs:GetCoverage></ctl:body>
-							<ctlp:XMLValidatingParser>
-								<ctlp:schemas>
-									<ctlp:schema type="url">http://schemas.opengis.net/wcs/2.0/wcsAll.xsd</ctlp:schema>
-								</ctlp:schemas>
-							</ctlp:XMLValidatingParser>
+							<ctlp:HTTPParser>
+							    <ctlp:parse>
+							        <ctlp:NullParser />
+							    </ctlp:parse>
+							</ctlp:HTTPParser>
 						</ctl:request>
 					</xsl:variable>
-					<xsl:choose>
-						<xsl:when test="empty($GetCoverage)">
-							<ctl:message>[FAILURE] Parsing or validation errors.</ctl:message>
-							<ctl:fail/>
-						</xsl:when>
-						<xsl:otherwise>
-							<xsl:if test="not(empty($GetCoverage//*[local-name()='Exception']))">
-								<ctl:message>[FAILURE] The result should not be an ows:ExceptionReport.</ctl:message>
-								<ctl:fail/>
-							</xsl:if>
-						</xsl:otherwise>
-					</xsl:choose>
+					<xsl:variable name="type">
+					    <xsl:value-of select="$GetCoverage/response/content/response/headers/header[@name='Content-Type']" />
+					</xsl:variable>
+					<xsl:if test="not(contains('image/', $type)) and $type != 'application/x-hdf5'">
+					    <ctl:message>[FAILURE] The result should be an image/* or application/x-hdf5</ctl:message>
+					    <ctl:fail />
+					</xsl:if>
 					<xsl:variable name="GetCoverage">
 						<ctl:request>
 							<ctl:url>

